sw/airborne/modules/computer_vision/lib/vision/image.h

```diff
@@ -115,4 +115,7 @@ void pyramid_next_level(struct image_t *input, struct image_t *output, uint8_t b
 void pyramid_build(struct image_t *input, struct image_t *output_array, uint8_t pyr_level, uint16_t border_size);
 void image_gradient_pixel(struct image_t *img, struct point_t *loc, int method, int *dx, int *dy);
 
+/* for minidrone */
```

Comment:
This description is lacking, can you give it a more generic and descriptive message, also, where is this function defined, can't find it in this PR.